src/azure-cli/azure/cli/command_modules/appservice/_validators.py

```diff
@@ -25,9 +25,11 @@ def validate_site_create(cmd, namespace):
             and isinstance(namespace.plan, str):
         resource_group_name = namespace.resource_group_name
         plan = namespace.plan
+        print(plan)
```

Comment:
logger is better

Reply:
Thanks for pointing this out - I had this while I was trying to debug. Will remove this one.